### PR TITLE
Added trip tests as xfails

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1155

### CODE REVIEW

File: artemis/tests/fixtures/trip_add_new_trip_151515.json [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+        "pointFinUtilisationSillon": {
+          "@id": "0f4666b5-082f-4006-9e06-65702e45fdda"
+        },
+        "porteur": true,
+        "rangPRDebutUtilisationSillon": 1,
+        "decalagePasseMinuit": 0,
+        "rangPRFinUtilisationSillon": 210
+      }
+    ],
+    "numeroCourse": "151515",
```
Comment (pbougue) [inline artemis/tests/fixtures/trip_add_new_trip_151515.json]: maybe a bit overkill for first test: we might put a train number already used to check there's no conflict...
Maybe worth a JIRA ticket.

File: artemis/tests/guichet_unique_test.py [hunk 2]
```diff
@@ -1135,6 +1134,63 @@ def test_kirin_cots_sequence_09(self):
...
+        Test an added circulation
+
+        Requested datetime: 2012/11/20 12:00:00
+        From: gare de Auxerre-St-Gervais
+        To:   gare de Marseille-St-Charles
+
+        Before added circulation, no solution can be found without transfer.
+        After added circulation, a train travels from 13:00:00 to 17:00:00
+        """
+        self.send_and_wait('trip_base_add_new_full_trip.json')
```
Comment (pbougue) [inline artemis/tests/guichet_unique_test.py]: This file isn't provided, is it on purpose (waiting to know details about it's content)?

File: artemis/tests/guichet_unique_test.py [hunk 3]
```diff
@@ -1135,6 +1134,63 @@ def test_kirin_cots_sequence_09(self):
...
+                     data_freshness="base_schedule")
+
+        self.journey(_from="stop_area:OCE:SA:87683573",
+                     to="stop_area:OCE:SA:87751008",
+                     datetime="20121120T120000",
+                     max_nb_transfers="0",
+                     data_freshness="realtime")
+
+    @xfail(reason="Waiting for fix - NAVP-1035", raises=AssertionError)
+    def test_kirin_cots_remove_added_course(self):
```
Comment (pbougue) [inline artemis/tests/guichet_unique_test.py]: For test time, we should probably merge those 2 tests (if test above is actually using the same file, not sure)

File: artemis/tests/fixtures/trip_add_new_trip_151515.json [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+        "cr": "0087",
+        "ci": "683573",
+        "ch": "BV",
+        "timeZoneLocale": "Europe/Paris",
+        "horaireVoyageurArrivee": {
+          "dateHeure": "2012-11-20T12:00:00+0000",
+          "heureLocale": "13:00:00",
+          "idFuseauHoraire": "Europe/Paris",
+          "nbJoursPasseMinuit": 0,
+          "statutCirculationOPE": "CREATION"
```
Comment (pbougue) [inline artemis/tests/fixtures/trip_add_new_trip_151515.json]: So we have AJOUTEE on trip and CREATION on stop.
Makes sense, but should we check that SNCF does it this way?
(same goes for SUPPRIMEE and SUPPRESSION).

File: artemis/tests/guichet_unique_test.py (hunk 2 above)
Comment (mbounabi) [reply artemis/tests/guichet_unique_test.py]: No, it was the previous name of the file 'trip_add_new_trip_151515'.

File: artemis/tests/fixtures/trip_add_new_trip_151515.json (hunk 1 above)
Comment (mbounabi) [reply artemis/tests/fixtures/trip_add_new_trip_151515.json]: this ticket is about simple cases

File: artemis/tests/guichet_unique_test.py (hunk 3 above)
Comment (mbounabi) [reply artemis/tests/guichet_unique_test.py]: I don't think it's a good idea, as it is clearly 2 separate use cases.

File: artemis/tests/fixtures/trip_add_new_trip_151515.json (hunk 4 above)
Comment (mbounabi) [reply artemis/tests/fixtures/trip_add_new_trip_151515.json]: We already checked with the MATRIX tool.

File: artemis/tests/guichet_unique_test.py [hunk 5]
```diff
@@ -1135,6 +1134,63 @@ def test_kirin_cots_sequence_09(self):
...
+        """
+        Test the removal of an added circulation
+
+        Requested datetime: 2012/11/20 12:00:00
+        From: gare de Auxerre-St-Gervais
+        To:   gare de Marseille-St-Charles
+
+        Before added circulation, no solution can be found without transfer.
+        After added circulation, a train travels from 13:00:00 to 17:00:00
```
Comment (pbougue) [inline artemis/tests/guichet_unique_test.py]: This comment is not up to date (testing removal here)

File: artemis/tests/guichet_unique_test.py (hunk 3 above)
Comment (pbougue) [reply artemis/tests/guichet_unique_test.py]: OK, in that case the proposition I made to remove one test after @xlqian 's PR is not worthy either, I guess.
Those 2 situations are about a sequence containing one-another and if the shortest one (simplest one) breaks, the longest does too.
The main point being to save a kirin db purge (done at start of each test method) in both situations.

Let's not do the removal for the other situation if we decide to merge this PR as-is, to be consistent.

File: artemis/tests/guichet_unique_test.py (hunk 5 above)
Comment (mbounabi) [reply artemis/tests/guichet_unique_test.py]: yes, but the course has to be added before being removed...

File: artemis/tests/fixtures/trip_add_new_trip_151515.json (hunk 1 above)
Comment (mbounabi) [reply artemis/tests/fixtures/trip_add_new_trip_151515.json]: @pbougue this case is noted in a ticket:
https://jira.kisio.org/browse/NAVP-1202

**Pull request merged**